src/command/shim.rs

```diff
@@ -152,13 +163,18 @@ fn resolve_shim(session: &mut Session, shim_name: &OsStr) -> Fallible<ShimKind>
     }
 }
 
+fn node_is_available(project: &Project, session: &Session) -> Fallible<Option<Version>> {
```

Comment:
Since this isn't a boolean predicate, maybe a better name would be something like `available_node_version`?

Reply:
Yeah I agree that's a better name